projects/storefrontapp-e2e-cypress/cypress/integration/regression/password-visibility/password-visibility.core-e2e-spec.ts

```diff
@@ -0,0 +1,37 @@
+import { viewportContext } from '../../../helpers/viewport-context';
+import { standardUser } from '../../../sample-data/shared-users';
+import { generateMail, randomString } from '../../../helpers/user';
+
+context('Password Visibility', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    beforeEach(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.clear();
+      });
+    });
+
+    it('should hide password by default', () => {
+      cy.visit('/login');
+      cy.get('button[aria-label="Show password"]').should('be.visible');
+    });
+
+    it('should show and hide password when toggled', () => {
+      cy.visit('/login');
+      cy.get('button[aria-label="Show password"]').click();
```

Comment:
should we add in this test case to check the password? What do you think?
I tried different cases on how to get the **** instead of the actual value when asserting in cypress, but it always gives me the actual value instead of **** 

You can try it out by doing

```
      cy.get('input[aria-label="Password"]').type('abc');
      cy.get('input[aria-label="Password"]').should('have.value', 'abc');
```